ownCloudAppShared/Client/Account/Controller/AccountController.swift

```diff
@@ -472,6 +475,11 @@ public class AccountController: NSObject, OCDataItem, OCDataItemVersioning, Acco
 				if let quickAccessFolderDataSource = specialItemsDataSources[.quickAccessFolder] {
 					sources.append(quickAccessFolderDataSource)
 				}
+				
+				if configuration.expandQuickAccess, let accountControllerSection = accountControllerSection,
+				   let expandedItemRefs = accountControllerSection.collectionViewController?.wrap(references: [  specialItemsDataReferences[.quickAccessFolder]! ], forSection: accountControllerSection.identifier) {
+					accountControllerSection.expandedItemRefs = expandedItemRefs
+				}
```

Comment:
Ideally `expandedItems` should not be modified or set directly, but via `.addExpanded()` and `.removeExpanded()`:

```suggestion
				if configuration.expandQuickAccess, let accountControllerSection = accountControllerSection,
				   let quickAccessItemRef = accountControllerSection.collectionViewController?.wrap(references: [  specialItemsDataReferences[.quickAccessFolder]! ], forSection: accountControllerSection.identifier).first {
					accountControllerSection.addExpanded(item: quickAccessItemRef)
				}
```